python/paddle/v2/fluid/executor.py

```diff
@@ -5,7 +5,13 @@
 
 
 class Executor(object):
-    def __init__(self, places):
+    def __init__(self, places=None):
+        if places is None:
+            if core.is_compile_gpu():
```

Comment:
I think we need to detect at runtime whether a GPU is present. It's very likely a compiled GPU version runs on a CPU only machine.